CoffeeShop_Pt1/CoffeeShop_Pt1/Drink.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <string>
+
+enum class DrinkType
+{
+	Tea,
+	Coffee
+};
+
+enum class Size
+{
+    S,
+    M,
+    L,
+    X,
+    XL
+};
+
+class Drink
+{
+public:
+    virtual void prepare() = 0;
+    virtual int getCost() = 0;
+    virtual ~Drink() {};
+};
+
+Drink* createDrink(DrinkType drinkType);
```

Comment:
just make it a static method of the Drink so it would be more clear where it belongs aka similar to namespace